sqlmesh/dbt/basemodel.py

```diff
@@ -140,8 +142,20 @@ def _validate_columns(cls, v: t.Any) -> t.Dict[str, ColumnConfig]:
 
     @property
     def all_sql(self) -> SqlStr:
+        return SqlStr("\n".join([self.hook_sql, self.sql_no_config]))
+
+    @property
+    def sql_no_config(self) -> SqlStr:
+        return SqlStr("")
+
+    @property
+    def sql_config(self) -> SqlStr:
         return SqlStr("")
 
+    @property
+    def hook_sql(self) -> SqlStr:
```

Comment:
Do we need this at all? It seems like we only use it in `all_sql` and we can just keep the previous version of it.

Reply:
encapsulated into all_sql